lib/handlers/pkg.put.js

```diff
@@ -187,6 +192,6 @@ const handler = async (sink, req, org, name, version) => {
         org,
     });
 
-    return await init(sink, incoming);
+    return init(sink, incoming);
```

Comment:
I am not sure about this one. I think this will trigger a difference in how Fastify is detecting if its handling a stream or a promise. We want the return value to be a stream, not a promise.

Reply:
I fixed this.